src/main/java/kitchenpos/domain/Menu.java

```diff
@@ -10,6 +11,17 @@ public class Menu {
     private Long menuGroupId;
     private List<MenuProduct> menuProducts;
 
+    public Menu() {
```

Comment:
```suggestion
    public Menu() {}
```
중요한 부분은 아니지만, [컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 작성해줘도 좋을 것 같습니다..!

Reply:
인텔리제이의 code reformatting 기능을 사용하니 자동으로 그렇게 수정해줘서 신경을 안썼었네요!
[빈블럭](https://github.com/JunHoPark93/google-java-styleguide#414-%EB%B9%88-%EB%B8%94%EB%9F%AD%EB%93%A4-%EC%95%84%EB%A7%88-%EA%B0%84%EA%B2%B0%ED%95%98%EA%B2%8C)의 경우 줄바꿈하거나 하지않는 경우 둘 다 허용하는 걸로 보이는데 제시해주신 방법이 깔끔해보여서 빈블럭들 찾아서 수정해봤습니다!